concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -166,6 +166,21 @@ impl fmt::Display for PacketType {
     }
 }
 
+impl PacketType {
+    /// Determine whether a packet type can be relayed.
+    /// Those that are must be subject to appropriate de-duplication
+    /// checks to ensure they are not relayed endlessly.
+    pub fn is_rebroadcastable(&self) -> bool {
+        match self {
+            PacketType::Block => true,
+            PacketType::Transaction => true,
+            PacketType::FinalizationRecord => true,
+            PacketType::FinalizationMessage => true,
+            PacketType::CatchUpStatus => false,
+        }
```

Comment:
Maybe add
`_ => false,`?

Reply:
> Maybe add
> `_ => false,`?

The list already contains all the different package types, so this does not add anything. The compiler checks completeness of the patterns.

Reply:
I think it's better not to have a default case here, so that if we add a new message type then the compiler will require us to be explicit about whether it should be rebroadcastable.